flink-table/flink-table-planner/src/test/resources/jsonplan/testGetJsonPlan.out

```diff
@@ -2,7 +2,7 @@
    "flinkVersion":"",
    "nodes":[
       {
-         "class":"org.apache.flink.table.planner.plan.nodes.exec.stream.StreamExecTableSourceScan",
+         "context": "1_stream-exec-table-source-scan_1",
```

Comment:
call it `id` independent of our discussion outcome

Reply:
You mean instead of `context`?